pkgs/tools/misc/calamares-nixos-extensions/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "calamares-nixos-extensions";
+  version = "0.0.1";
+
+  src = fetchurl {
+    url = "https://github.com/vlinkz/${pname}/archive/refs/tags/${version}.tar.gz";
+    sha256 = "sha256-Ua7NJXwu9QsWrTzRKMPQnlkr7lECBNAKBRjD6bwCHa8=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/{lib,share}/calamares
+    cp -r modules $out/lib/calamares/
+    cp -r config/* $out/share/calamares/
+    cp -r branding $out/share/calamares/
+  '';
+
+  meta = with lib; {
+    description = "Calamares modules for NixOS";
+    license = with licenses; [ gpl3Plus bsd2 ];
+    platforms = platforms.linux;
```

Comment:
Missing maintainer and homepage.

Reply:
For the maintainer, I'm unsure who to add. Should I just add myself?

Reply:
if you want to maintain it, sure.

for such an important component it would be good to have at least 2 maintainers. maybe we can find an additional maintainer

Reply:
Given this [OfBorg output](https://gist.github.com/GrahamcOfBorg/5a9e20a3f580da0751ac6fd48b9d99bc), do I need to add myself somewhere else as well?

Reply:
Yes, in https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix.